src/App.jsx

```diff
@@ -1,5 +1,9 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
+import { GoodList } from './components/GoodList/GoodList';
+import { Reset } from './components/Reset/Reset';
```

Comment:
There is a typo in the import path for the `Reset` component. The folder name `components` is misspelled as `compotents`. Please correct it to `components`.